go/ephemeral/user_ek.go

```diff
@@ -288,7 +298,39 @@ func getLatestUserEKMetadataMaybeStale(ctx context.Context, g *libkb.GlobalConte
 		return nil, err
 	}
 
-	return metadata, nil
+	return statement, nil
+}
+
+func getLatestUserEKMetadataMaybeStale(ctx context.Context, g *libkb.GlobalContext) (metadata *keybase1.UserEkMetadata, err error) {
+	defer g.CTrace(ctx, "getLatestUserEKMetadataMaybeStale", func() error { return err })()
+
+	statement, err := getLatestUserEKStatementMaybeStale(ctx, g)
+	if err != nil || statement == nil {
+		return nil, err
+	}
+
+	return &statement.CurrentUserEk, nil
+}
+
+func getAllActiveUserEKs(ctx context.Context, g *libkb.GlobalContext, now keybase1.Time) (usereks []keybase1.UserEkMetadata, err error) {
```

Comment:
let's pass in `merkleRoot` here so we do the time conversion once instead of for every caller? it's more consistent with the other parts of the code that accept this and forces us to take the time from the merkleRoot instead of whatever the caller decides. what do you think?